src/main/resources/org/jenkinsci/plugins/authorizeproject/form/dropdownDescriptorSelector.jelly

```diff
@@ -66,7 +66,7 @@ THE SOFTWARE.
       <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
                            selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}"
                            staplerClass="${descriptor.clazz.name}"
-                           lazy="${attrs.lazy?attrs.lazy+',':''}descriptor,it">
+                           lazy="${(attrs.capture != null and attrs.capture != '')?attrs.capture+',':''}descriptor,it">
```

Comment:
Hmm...from this document, `value?true:false` should return `true`.
http://docs.groovy-lang.org/latest/html/documentation/index.html#Groovy-Truth

I have to have a further investigation.